app/controllers/homepage_controller.rb

```diff
@@ -37,20 +37,38 @@ def index
 
     listings_per_page = APP_CONFIG.grid_listings_limit
 
+    filter_params = {}
+
+    # :transaction_type was renamed to :order_type
```

Comment:
The risk here is we haven't yet locked the order_type term so this might deprecate next week when the design of shape ui progresses and we define the term to go forward with. Order type is the best candidate for now but not yet decided.